bundles/org.openhab.binding.irobot/src/main/java/org/openhab/binding/irobot/internal/handler/RoombaHandler.java

```diff
@@ -12,7 +12,36 @@
  */
 package org.openhab.binding.irobot.internal.handler;
 
-import static org.openhab.binding.irobot.internal.IRobotBindingConstants.*;
+import static org.openhab.binding.irobot.internal.IRobotBindingConstants.BIN_FULL;
+import static org.openhab.binding.irobot.internal.IRobotBindingConstants.BIN_OK;
+import static org.openhab.binding.irobot.internal.IRobotBindingConstants.BIN_REMOVED;
+import static org.openhab.binding.irobot.internal.IRobotBindingConstants.BOOST_AUTO;
+import static org.openhab.binding.irobot.internal.IRobotBindingConstants.BOOST_ECO;
+import static org.openhab.binding.irobot.internal.IRobotBindingConstants.BOOST_PERFORMANCE;
+import static org.openhab.binding.irobot.internal.IRobotBindingConstants.CHANNEL_ALWAYS_FINISH;
+import static org.openhab.binding.irobot.internal.IRobotBindingConstants.CHANNEL_BATTERY;
+import static org.openhab.binding.irobot.internal.IRobotBindingConstants.CHANNEL_BIN;
+import static org.openhab.binding.irobot.internal.IRobotBindingConstants.CHANNEL_CLEAN_PASSES;
+import static org.openhab.binding.irobot.internal.IRobotBindingConstants.CHANNEL_COMMAND;
+import static org.openhab.binding.irobot.internal.IRobotBindingConstants.CHANNEL_CYCLE;
+import static org.openhab.binding.irobot.internal.IRobotBindingConstants.CHANNEL_EDGE_CLEAN;
+import static org.openhab.binding.irobot.internal.IRobotBindingConstants.CHANNEL_ERROR;
+import static org.openhab.binding.irobot.internal.IRobotBindingConstants.CHANNEL_LAST_COMMAND;
+import static org.openhab.binding.irobot.internal.IRobotBindingConstants.CHANNEL_PHASE;
+import static org.openhab.binding.irobot.internal.IRobotBindingConstants.CHANNEL_POWER_BOOST;
+import static org.openhab.binding.irobot.internal.IRobotBindingConstants.CHANNEL_RSSI;
+import static org.openhab.binding.irobot.internal.IRobotBindingConstants.CHANNEL_SCHEDULE;
+import static org.openhab.binding.irobot.internal.IRobotBindingConstants.CHANNEL_SCHED_SWITCH;
+import static org.openhab.binding.irobot.internal.IRobotBindingConstants.CHANNEL_SCHED_SWITCH_PREFIX;
+import static org.openhab.binding.irobot.internal.IRobotBindingConstants.CHANNEL_SNR;
+import static org.openhab.binding.irobot.internal.IRobotBindingConstants.CMD_CLEAN;
+import static org.openhab.binding.irobot.internal.IRobotBindingConstants.CMD_CLEAN_REGIONS;
+import static org.openhab.binding.irobot.internal.IRobotBindingConstants.CMD_DOCK;
+import static org.openhab.binding.irobot.internal.IRobotBindingConstants.CMD_PAUSE;
+import static org.openhab.binding.irobot.internal.IRobotBindingConstants.CMD_STOP;
+import static org.openhab.binding.irobot.internal.IRobotBindingConstants.PASSES_1;
+import static org.openhab.binding.irobot.internal.IRobotBindingConstants.PASSES_2;
+import static org.openhab.binding.irobot.internal.IRobotBindingConstants.PASSES_AUTO;
```

Comment:
Why? Or does your IDE do it automatically ?

Reply:
Oh I did not recognize that. The imports were automatically managed by my IDE. I have reverted the static imports to the original one.